meta-lmp-base/recipes-security/optee/optee-client-fio.inc

```diff
@@ -5,13 +5,15 @@ HOMEPAGE = "https://www.op-tee.org/"
 LICENSE = "BSD-2-Clause"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=69663ab153298557a59c67a60a743e5b"
 
-inherit systemd cmake pkgconfig
+inherit systemd cmake pkgconfig useradd_base
```

Comment:
I have found some little differences from the one in meta-arm
```patch
-inherit systemd cmake pkgconfig useradd_base 
- 
-DEPENDS = "util-linux-libuuid" 
+inherit systemd update-rc.d cmake useradd 
```

Reply:
I dont think we want to change the CFG_TEE_FS_PARENT_PATH. And we need uuid for sysroot AFAIR.
I tried a clean local build with useradd and still wont work for me. maybe I need to clear the cache...

Reply:
to fix the version issue another patch is needed
```patch
--- a/meta-lmp-base/files/lmp-group-table
+++ b/meta-lmp-base/files/lmp-group-table
@@ -50,6 +50,7 @@ shutdown:x:70:
 nobody:*:99:
 users:x:100:
 pulse:x:171:
+teeclnt:x:967:
 ptest:x:968:
 sgx:x:969:
 lpadmin:x:970:

```

Reply:
amazing! I was wondering how it could work without that group - I dont see it created and yet the service was being started. ok thanks, I'll fix.